src/shared/seccomp-util.c

```diff
@@ -753,6 +777,17 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "swapoff\0"
                 "swapon\0"
         },
+        [SYSCALL_FILTER_SET_SYNC] = {
+                .name = "@sync",
+                .help = "Synchronize files and memory to storage",
+                .value =
+                "fdatasync\0"
+                "fsync\0"
+                "msync\0"
+                "sync\0"
+                "sync_file_range\0"
+                "syncfs\0"
+        },
```

Comment:
Maybe prefix this set with `fs-sync` or `file-system-sync` ?

Reply:
dunno if that's necessary... This is in the context of syscalls after all, and in syscalls the name "sync" is clear enough i think.